### ISSUE TITLE
[FE] My VA: Update "Next Appointment" widget header

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#24906]
The h4 should be an h3.

![Screen Shot 2021-05-18 at 2.58.38 PM.png](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/19a6a49d-8f52-4ccf-a952-4f3d425f072b)

**Comment by erikphansen** [comment]
I've validated this on staging. The Next Appointment heading is now an H3 as it should be.

![headers.gif](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/537e9d93-f9c6-4740-be77-ebcb37aebe89)
